### PR TITLE
[CloudPC] Get Supported CloudPc Remote Actions

### PR DESCRIPTION
**Pull request by jannyma**

### CODE REVIEW

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 1]
```diff
@@ -2556,23 +2556,41 @@
...
+        "ChangeList": [
+          {
             "Id": "f3def7ad-015a-48bc-aaf2-b944069124cd",
-            "Cloud": "Prod",
-            "Version": "beta",
-            "CreatedDateTime": "2022-08-24T02:53:28.597151Z",
-            "WorkloadArea": "Devices and apps",
-            "SubArea": "Cloud PC"
-        }
+            "ApiChange": "Method",
```
Comment (RuiHou105) [inline changelog/Microsoft.CloudManagedDesktop.json:2563]: The changelog didn't  include all changes. Please also add changelog for: new resource `cloudPcRemoteActionCapability`, new enum `actionCapability `and `cloudPcRemoteActionName`

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
@hanky0301 
1.The format of this PR doesn't follow the standard one. e.g., the headers should start with ## or ###...So could you please follow this [example ](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18829)to update yours?
2.As you add a new API, please also update the toc.yml file, like the example above does. 
3.Please create a roadmap workitem and attach the workitem number in comment. Here is an [example](https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/14460/). If no access, please do these two steps firstly.
- Request access via [Improved ADO Access - 1ES Wikirequest to join](https://www.1eswiki.com/wiki/Improved_ADO_Access) .
- Join the Microsoft Graph Roadmap Users ([graphroadmap@microsoft.com](mailto:graphroadmap@microsoft.com)) security group on [idweb](https://idwebelements/GroupManagement.aspx?Group=graphroadmap&Operation=join) .

File: changelog/Microsoft.CloudManagedDesktop.json (hunk 1 above)
Comment (hanky0301) [reply changelog/Microsoft.CloudManagedDesktop.json]: Done

**Comment by jannyma** [comment]
Roadmap work item no: 15657
https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/15657

File: api-reference/beta/toc.yml [hunk 2]
```diff
@@ -10394,6 +10394,8 @@ items:
             href: api/cloudpc-endgraceperiod.md
           - name: Get connectivity history
             href: api/cloudpc-getcloudpcconnectivityhistory.md
+          - name: Get supported remote actions
```
Comment (RuiHou105) [inline api-reference/beta/toc.yml:10397]: @hanky0301  Please place this API after `Get review status`, as all APIs should be sorted alphabetically.

**Comment by RuiHou105** [comment]
According to the [build failure](https://dev.azure.com/msrobots/apidoctor/_build/results?buildId=102483&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=dac0b392-2308-5704-b9f7-c5b20adf012e), there are two errors. So could you please help fix them? Thanks.

image

File: api-reference/beta/toc.yml (hunk 2 above)
Comment (hanky0301) [reply api-reference/beta/toc.yml:10397]: Will fix it right away

**Comment by Jarbas-MSFT** [comment]
@Lauragra, this PR is ready for a final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**